mailpoet/tasks/phpstan/phpstan.neon

```diff
@@ -42,12 +42,14 @@ parameters:
     - '/Parameter #1 \$cssOrXPath of method AcceptanceTester::moveMouseOver\(\) expects string\|null, array<string, string> given./'
     - '/Function expect invoked with 1 parameter, 0 required\./'
     - '/Call to method getName\(\) on an unknown class _generated\\([a-zA-Z])*Cookie/' # codeception generate incorrect return type in ../../tests/_support/_generated
+    - '/Call to static method container\(\) on an unknown class/'
```

Comment:
Our current version of the [woocommerce-stubs](https://github.com/php-stubs/woocommerce-stubs) do not know the new class. I tried upgrading our package, which would solve this problem, but this resulted in many other unrelated problems, I couldn't figure out quickly. Therefore I just ignore the problem for now.

Any idea?

Reply:
Maybe we can update the woocommerce-stubs in a different task. What do you think?

Reply:
Will create an issue.